dashboard.yml

```diff
@@ -73,9 +73,10 @@
 
 # using groups[] here otherwise it can't fallback to the mon if there's no mgr group.
 # adding an additional | default(omit) in case where no monitors are present (external ceph cluster)
-- hosts: "{{ groups[mgr_group_name] | default(groups[mon_group_name]) | default(omit) }}"
+- hosts: "{{ groups[grafana_server_group_name] }}"
```

Comment:
Don't change this. The Ceph Dashboard is deployed on the mgrs or mons nodes not on the nodes present on the grafana-server group

Reply:
Restored